src/components/UI/Icon/Icon.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { SVGProps } from "react";
 import { CheckCircle } from "react-feather";
+import warningIcon from "../../../static/images/Alert/warning.png";
 
```

Comment:
if only can use `AlertCircle` lol, next time u can counter propose design to keep maintenance sane